### ISSUE TITLE
Perpetual Hauling

### ISSUE THREAD

**Issue by Shadezy** [FluffierThanThou/StackMerger#2]
[output_log.txt](https://github.com/FluffierThanThou/StackMerger/files/754339/output_log.txt)

This is the output log you asked for. I'll just copy and paste what I already posted on the workshop page just so it's here as well.

I have a problem with colonists perpetually hauling materials. I have two stacks right next to eachother and one is maxed out (I believe, not entirely sure on the numbers). The colonist will just keep going back and forth putting some of the stack into the other and then back and they get caught in a loop. I have narrowed it down to this mod specifically and have some ideas of mods that might be conflicting. I have increased Stack and Stack XXL which change the values of max stacks. And Extended Storage which adds constructible areas to place materials in. I assume it's the last mod I mentioned that is affecting it, but since it adds actual constructible elements in the game, and I have already built them, I haven't tried to disable it to test.

**Comment by FluffierThanThou** [comment]
could you confirm that this still happens on the latest version? It should have been fixed.

**Comment by Shadezy** [comment]
It didn't work at first, but then I tried deconstructing and rebuilding the storage containers and it seems to be fixed now. The problem only happened when there were two very big stacks next to eachother (happened with my steel and muffalo leather specifically), but when I deconstructed the containers, both stacks were automatically consolidated so when they were rebuilt, they fit neatly on one side. All in all, I don't think it is problem with your mod anymore, if there still is a problem. Thank you, and sorry for using up a lot of your time.
